en/material-system/overview.md

```diff
@@ -1,5 +1,14 @@
 # Material System Overview
 
+## Material System Upgrade Guide
+
+Cocos Creator has supported the material system since v2.x. However, we are continuously improving the design of the material system and the built-in Shader API, so there may be some adjustments in the upgrade from v2.x to v3.0 and subsequent upgrades. The Effect asset does not currently support automatic upgrades, so please refer to the following documentation to upgrade:
```

Comment:
```suggestion
Cocos Creator has supported the material system since v2.x. However, the design of the material system and the built-in Shader API are continually being improved, there may be some adjustments in the upgrade from v2.x to v3.0 and subsequent upgrades. The Effect asset does not currently support automatic upgrades, so please refer to the following documentation to upgrade:
```